tests/Integration/User/Application/EventSubscriber/PasswordChangedEventSubscriberTest.php

```diff
@@ -26,7 +26,7 @@ protected function setUp(): void
 
     public function testConfirmationEmailSent(): void
     {
-        $emailAddress = $this->faker->email();
+        $emailAddress = 'test@example.com';
```

Comment:
We should use faker instead of hardcoding the test data
It will be better to return back this line